routes/web.php

```diff
@@ -12,9 +12,9 @@
 */
 
 Route::get('/', function () {
```

Comment:
Usar função na rota é muito chato, invalida a criação de cache das rotas e fica meio anti-pattern, talvez ter um controller como aquele seria uma ideia melhor

Reply:
Ah sim esqueci de tirar isso, coloquei só porque o `HomeController` está com o `auth`. Assim que esta PR for aprovada eu tiro isso na hora do merge.

Reply:
Top